neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -30,6 +30,24 @@ protected StorageKey CreateAccountKey(UInt160 account)
             return CreateStorageKey(Prefix_Account, account);
         }
 
+        protected override long GetPriceForMethod(string method)
+        {
+            switch (method)
+            {
+                case "name":
+                case "symbol":
+                case "decimals":
+                    return 0;
```

Comment:
At least the same as NOP

Reply:
https://github.com/neo-project/neo/blob/1a415b48c48e06fd8f167e7d02d92febe73b0844/neo/SmartContract/InteropService.cs#L56

Every contract invocation has a base fee.